constants/src/block.rs

```diff
@@ -0,0 +1,25 @@
+//! Block related.
+
+use core::time::Duration;
+
+use crate::macros::monero_definition_link;
+
+/// The maximum block height possible.
+#[doc = monero_definition_link!(a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623, "/src/cryptonote_config.h", 40)]
+pub const MAX_BLOCK_HEIGHT: u64 = 500_000_000;
+
+/// [`MAX_BLOCK_HEIGHT`] as a [`usize`].
+#[expect(clippy::cast_possible_truncation, reason = "will not be truncated")]
+pub const MAX_BLOCK_HEIGHT_USIZE: usize = MAX_BLOCK_HEIGHT as usize;
+
+/// Target block time for hardfork 1.
+#[doc = monero_definition_link!(a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623, "/src/cryptonote_config.h", 81)]
+///
+/// ref: <https://monero-book.cuprate.org/consensus_rules/blocks/difficulty.html#target-seconds>
+pub const BLOCK_TIME_V1: Duration = Duration::from_secs(60);
+
+/// Target block time from hardfork v2.
+#[doc = monero_definition_link!(a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623, "/src/cryptonote_config.h", 80)]
+///
+/// ref: <https://monero-book.cuprate.org/consensus_rules/blocks/difficulty.html#target-seconds>
+pub const BLOCK_TIME_V2: Duration = Duration::from_secs(120);
```

Comment:
Used in `types/src/hard_fork.rs`.

I think this also needs to be used here? https://github.com/Cuprate/cuprate/pull/300/commits/7902f0a902f1caece3a1a12e268ef97fe13df59a#diff-bdf64dc01a71e5057249b389c20030fd2f8db7be263cee26318bd976500f075dL112-R114